jax/_src/lax/lax.py

```diff
@@ -4681,53 +4711,92 @@ def _scatter_jvp(primals, tangents, *, update_jaxpr, update_consts,
   partial(_scatter_batching_rule, scatter))
 
 
-def _reduce_shape_rule(operand, init_value, *, computation, jaxpr, consts,
-                       dimensions):
-  return tuple(np.delete(operand.shape, dimensions))
+def _reduce_abstract_eval_rule(*avals, **params):
```

Comment:
Do we have to use `**params` here or can we revert to using keyword arguments?

Reply:
Done.